homework/Contributor.js

```diff
@@ -12,8 +12,31 @@ class Contributor {
    * Render the contributor info to the DOM.
    * @param {HTMLElement} container The container element in which to render the contributor.
    */
-  render(container) {
+  static render(data, container) {
```

Comment:
Please make this an `instance` method of the class rather than `static`. 🙂

Reply:
Done.